code/game/turfs/simulated/floor.dm

```diff
@@ -42,6 +42,12 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	if(floor_tile)
 		builtin_tile = new floor_tile
 
+/turf/simulated/floor/Destroy()
+	if(builtin_tile)

```

Comment:
You guys may want to address the 100+ other "If(thing) then qdel(thing)" in the code then, as they're still being added quite frequently; the atmospherics overhaul of late, by Duncan, did just this in a number of place.